### PR TITLE
Adding component and sub component tests

### PR DESCRIPTION
**Pull request by amitkrout**

### CODE REVIEW

File: ci-operator/config/redhat-developer/odo/redhat-developer-odo-master.yaml [hunk 1]
```diff
@@ -5,7 +5,6 @@ resources:
     limits:
       memory: 6Gi
     requests:
-      cpu: 100m
```
Comment (amitkrout) [inline ci-operator/config/redhat-developer/odo/redhat-developer-odo-master.yaml]: @mohammedzee1000 Removed as discussed in our bjn call with @kadel

Comment (mohammedzee1000) [reply ci-operator/config/redhat-developer/odo/redhat-developer-odo-master.yaml]: did you tru to update pr from a branch that was already used without rebasing

Comment (amitkrout) [reply ci-operator/config/redhat-developer/odo/redhat-developer-odo-master.yaml]: no, Anyway now its fine.

**Comment by mohammedzee1000** [comment]
Ah the missing charactere

**Comment by mohammedzee1000** [comment]
Oh we have rehearsals for that now. Awesome. On flip side, we are going to have to wait

**Comment by amitkrout** [comment]
Seems ```90 sec``` is not enough - https://github.com/redhat-developer/odo/blob/master/tests/e2e/component.go#L90. Increasing it to 180 sec for worst case scenario in odo master.

**Pull request merged**